bounties/npm/scalajs-standalone-bin/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-17",
+	"AffectedVersionRange": "*",
+	"Summary": "Resources Downloaded over Insecure Protocol",
+	"Contributor": {
+		"Discloser": "",
+		"Fixer": ""
+	},
+	"Package": {
+		"Registry": "npm",
+		"Name": "scalajs-standalone-bin",
+		"URL": "https://www.npmjs.com/package/scalajs-standalone-bin",
+		"Downloads": "1205"
+	},
+	"CWEs": [
+		{
+			"ID": "CWE-494",
+			"Description": ""
+		}
+	],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "N",
+		"AC": "H",
+		"PR": "L",
+		"UI": "R",
+		"S": "U",
+		"C": "H",
+		"I": "H",
+		"A": "H",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "7.1"
+	},
+	"CVEs": [
+		"CVE-2016-10634"
+	],
+	"Repository": {
+		"URL": "https://github.com/k-kinzal/scalajs-standalone-bin",
+		"Codebase": [
+			"JavaScript"
+		]
+	},
+	"Permalinks": [
+		"https://github.com/k-kinzal/scalajs-standalone-bin/blob/0298772d1e1cad8bfabd56ec901dc385f0805a6d/src/common.js#L12"
```

Comment:
Change with https://github.com/k-kinzal/scalajs-standalone-bin/blob/master/src/common.js#L12

Reply:
Got you, cheers 👍🏼